Makefile

```diff
@@ -25,3 +29,5 @@ release: all
 	git commit -m 'Update restylers.yaml'
 	git tag -s -m "$(RELASE_TAG)" "$(RELASE_TAG)"
 	git push --follow-tags
+
+.SECONDARY:
```

Comment:
What's this do?

Reply:
see the commit message for it: https://github.com/restyled-io/restylers/pull/34/commits/916bef81cb6b38ac9050f00e3bd1f0a3b08c5d6f